.drone.yml

```diff
@@ -26,7 +26,7 @@ steps:
       filename: unit-tests-cache.tar
 
   - name: unit-tests
-    image: codedotorg/code-dot-org:latest
+    image: codedotorg/code-dot-org:1.2
```

Comment:
We also need to update the Image version specified for the User Interface container (line 123).